ramls/userTenant.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "User Tenant Association Schema",
+  "type": "object",
+  "description": "An entity that describes an association between a user and a tenant in a consortium",
+  "properties": {
+    "id": {
+      "description": "identifier",
+      "type": "string"
+    },
+    "userId": {
+      "description": "User identifier",
+      "type": "string"
+    },
+    "username": {
+      "description": "Username",
+      "type": "string"
+    },
+    "tenantId": {
+      "description": "Tenant identifier",
+      "type": "string"
+    },
+    "tenantName": {
+      "description": "Tenant name",
+      "type": "string"
+    },
+    "isPrimary": {
```

Comment:
All records will be primary records in mod-users so we don't need this field

Reply:
OK